src/reducers/index.test.ts

```diff
@@ -107,4 +109,33 @@ it('setCurrentMatchIndex', () => {
   };
   expect(reduce(initialState, action)).toEqual(expectedState);
 });
+
+it('updatePhoneNumberToContact', () => {
+  const initialState = storeStateDefault;
+  let { phoneNumberToContact, ...rest } = initialState;
+
+  let someContact: Contact = {
+    phoneNumber: '1234567890',
+    name: 'someName',
+    avatarImage: 'someImage'
+  };
+
+  let newPhoneNumberToContact: PhoneNumberToContact = {};
+  newPhoneNumberToContact['phoneNumber'] = someContact;
```

Comment:
newPhoneNumberToContact['phoneNumber'] = someContact;
to
newPhoneNumberToContact['+1234567890'] = someContact;